[2024-02-27T04:13:57.636+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.insert_into_table scheduled__2024-02-25T00:00:00+00:00 [queued]>
[2024-02-27T04:13:57.750+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v08.insert_into_table scheduled__2024-02-25T00:00:00+00:00 [queued]>
[2024-02-27T04:13:57.788+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-02-27T04:13:58.024+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): insert_into_table> on 2024-02-25 00:00:00+00:00
[2024-02-27T04:13:58.050+0000] {standard_task_runner.py:60} INFO - Started process 107 to run task
[2024-02-27T04:13:58.074+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v08', 'insert_into_table', 'scheduled__2024-02-25T00:00:00+00:00', '--job-id', '497', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpa55iqro0']
[2024-02-27T04:13:58.081+0000] {standard_task_runner.py:88} INFO - Job 497: Subtask insert_into_table
[2024-02-27T04:13:58.224+0000] {task_command.py:423} INFO - Running <TaskInstance: dag_with_postgres_operator_v08.insert_into_table scheduled__2024-02-25T00:00:00+00:00 [running]> on host f592a0b4b395
[2024-02-27T04:13:58.553+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Shiv' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v08' AIRFLOW_CTX_TASK_ID='insert_into_table' AIRFLOW_CTX_EXECUTION_DATE='2024-02-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-25T00:00:00+00:00'
[2024-02-27T04:13:58.554+0000] {sql.py:276} INFO - Executing: 
            INSERT INTO dag_runs (dt, dag_id) VALUES ('2024-02-25', 'dag_with_postgres_operator_v08');
        
[2024-02-27T04:13:58.577+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-27T04:13:58.634+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-27T04:13:58.660+0000] {sql.py:450} INFO - Running statement: 
            INSERT INTO dag_runs (dt, dag_id) VALUES ('2024-02-25', 'dag_with_postgres_operator_v08');
        , parameters: None
[2024-02-27T04:13:58.663+0000] {sql.py:459} INFO - Rows affected: 1
[2024-02-27T04:13:58.692+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v08, task_id=insert_into_table, execution_date=20240225T000000, start_date=20240227T041357, end_date=20240227T041358
[2024-02-27T04:13:58.778+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-27T04:13:58.832+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
